source/ref_batch.rst

```diff
@@ -18,7 +18,7 @@ Currently Batch Inference supports the following ways to delivery data for forec
 
 In future we consider to add ability to process data directly from Kafka topic and other async data sources.
 
-Please also take a look at `example <https://github.com/odahu/odahu-examples/tree/feat/batch-inference-example/batch-inference>`_.
+Please also take a look at `example <https://github.com/odahu/odahu-examples/tree/1.5.0-rc4/batch-inference>`_.
```

Comment:
```suggestion
Please also take a look at `example <https://github.com/odahu/odahu-examples/tree/develop/batch-inference>`_.
```
Maybe, it's better to switch to `develop` (while our `master` branch fell behind in development) or switch to `master` as we are going to release this part of docs in 1.5.0.

Because, if we make any changes in batch-inference, we should update branch every time.

Reply:
Because, if we make **any changes** in batch-inference, we should update branch every time.

Why we should update branch every time?

Reply:
We cannot refer to not tagged version because, for example, it could lead to broken links or other inconsistent problems here.

So we will update this link every time when documentation will changed, otherwise we should refer to some predictable revision in odahu-examples repo